### PR TITLE
feat: power function added

### PR DESCRIPTION
**Pull request by vinusuhas-codecraft**

### CODE REVIEW

File: math.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+function powers(n,p){
```
Comment (FayazCodeCraft) [inline math.js]: Add Jsdoc for the function

**Review by FayazCodeCraft — CHANGES_REQUESTED** [review]

File: math.js (hunk 1 above)
Comment (ghost) [inline math.js]: change the variable names to be more descriptive

**Review by ghost — CHANGES_REQUESTED** [review]

File: math.js (hunk 1 above)
Comment (vinusuhas-codecraft) [reply math.js]: > Add Jsdoc for the function

ok will do it ASAP

File: math.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/**
+ *
```
Comment (FayazCodeCraft) [inline math.js]: Add description

**Review by FayazCodeCraft — CHANGES_REQUESTED** [review]

**Review by FayazCodeCraft — APPROVED** [review]
LGTM

**Review by ghost — APPROVED** [review]
- LGTM
- Squash your commits

**Pull request merged**